src/utils/oreToIron.ts

```diff
@@ -0,0 +1,8 @@
+export const oreToIron = (ore: bigint): number => {
+  return ore ? Number(ore) / 10000000 : 0
```

Comment:
This is the wrong divisor. One more zero is needed - 100000000

Generally, when doing wallet functionality you should be running the Iron Fish CLI and comparing against that. That is a good source of truth, and we are essentially just building a GUI for that functionality.

Reply:
Even better than knowing the right numbers here would be using the CurrencyUtils from the `ironfish` package. You can access this, I think?